README.md

```diff
@@ -0,0 +1,95 @@
+# TeleStore: Secure and Free Encrypted File Storage
+
+Effortlessly store and manage your files with robust encryption for ultimate privacy and security!
+
+[Report Bug](https://github.com/1mpossible-code/telegram-storage/issues) · [Request Feature](https://github.com/1mpossible-code/telegram-storage/issues)
+
+## Table of Contents
+
+- [About The Project](#about-the-project)
+- [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+
+## About The Project
+
+![ScreenRecording2024-06-01at8 24 39PM-ezgif com-video-to-gif-converter](https://github.com/1mpossible-code/telegram-storage/assets/109933928/6bf007f4-e813-4562-b2ca-7fc8cef585e6)
+
+TeleStore leverages the Telegram Bot API to store and retrieve files easily, allowing for a seamless file storage and management experience. Furthermore, TeleStore encrypts your files, ensuring your data is kept private. With its user-friendly interface and robust security measures, TeleStore is the perfect solution for managing your files securely and efficiently.
+
+
+## Built With
+
+This section lists any major frameworks/libraries used to bootstrap the project.
+
+- Telegram Bot API
+- Flask
+- Docker
+- React
+- Redux
+
+
+
+## Getting Started
+
+To get a local copy up and running, follow these simple steps.
+
+### Prerequisites
+
+- Docker 
+- [Telegram Bot](#https://core.telegram.org/bots/tutorial)
+
+### Installation
+1. **Clone the repository:**
+    ```bash
+    git clone git@github.com:1mpossible-code/telegram-storage.git
+    cd telegram-storage
+    ```
+
+2. **Start Docker containers:**
+    ```bash
+    docker-compose up
```

Comment:
add this to dependency: https://github.com/docker/compose

and update the command to `docker compose up`